play/src/lib/HelperFunctions.tsx

```diff
@@ -85,6 +86,50 @@ export const checkForSubmittedAnswerOnRejoin = (
   return { selectedAnswerIndex, isSubmitted };
 };
 
+/**
+ * on rejoining game, this checks if the player has already selected a confidence level
+ * @param hasRejoined - if a player is rejoining
+ * @param currentAnswer - the player's answer to the current question
+ * @param currentState - the current state of the game session
+ * @returns - the index of the confidence the player has submitted, null if they haven't selected a confidence, boolean to track submission
+ */
+export const checkForSelectedConfidenceOnRejoin = (
+  hasRejoined: boolean,
+  currentAnswer: ITeamAnswer | null | undefined,
+  currentState: GameSessionState
+): {
+  selectedConfidenceIndex: number | null;
+  isSelected: boolean;
+  timeOfLastSelect: number | null;
```

Comment:
hmm, is the purpose of having `null` is just because user hasn't selected anything? We can assign `0` if that's the case.

Reply:
Yes! That makes a lot more sense than representing time as null. I'll update all instances of timeOfLastSelect to address the above comments: (1) have the variable storing the `currentTime` in the `handleRadioChange` function in `ConfidenceMeter.tsx` document itself by renaming the var `currentEpochTimeInSeconds` and (2) change all typing of `timeOfLastSelect` from `number | null` to just `number`, giving it the value of 0 before the user interacts with the radio buttons